README.md

```diff
@@ -1,13 +1,11 @@
 # Project React Native App 📱
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This week we were to build an app using React Native. We should be using either a phone sensor like the accelerometer, an api-fetch or React Navigation. The app should be published using Expo.
```

Comment:
Good start of ReadMe - its really convenient to adress the goal that you do here - in order for the reviewer to be aware of the basic needs.